### PR TITLE
Add Templates Wizard and integration with Create Vm Wizard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: src/components/Wizard/CreateVmWizard/BasicSettingsTab.js [hunk 1]
```diff
@@ -150,19 +161,14 @@ export const getFormFields = (basicSettings, namespaces, templates, selectedName
...
+      disabled: userTemplate !== undefined,
     },
-    [START_VM_KEY]: {
-      id: 'start-vm',
-      title: 'Start virtual machine on creation',
-      type: 'checkbox',
-      noBottom: true,
-    },
+    [START_VM_KEY]: startVmCheckbox,
     /*
```
Comment (mareklibra) [inline src/components/Wizard/CreateVmWizard/BasicSettingsTab.js]: It's time to remove that ...

File: src/components/Wizard/CreateVmWizard/StorageTab.js [hunk 2]
```diff
@@ -14,140 +14,187 @@ import {
...
+  ATTACH_DISK_BUTTON,
+  CREATE_DISK_BUTTON,
   BOOTABLE_DISK,
   SELECT_BOOTABLE_DISK,
 } from './strings';
+import { STORAGE_TYPE_PVC, STORAGE_TYPE_DATAVOLUME, STORAGE_TYPE_REGISTRY } from './constants';
 
-const validatePvc = pvc => {
+const validateDataVolumeStorage = storage => {
   const errors = Array(4).fill(null);
```
Comment (mareklibra) [inline src/components/Wizard/CreateVmWizard/StorageTab.js]: For a follow-up: let's avoid hardcoding of indexes and their manipulation

File: src/components/Wizard/CreateVmWizard/StorageTab.js [hunk 3]
```diff
@@ -335,6 +393,18 @@ class StorageTab extends React.Component {
             .filter(pvc => !this.state.rows.some(row => row.name === pvc)),
           initialValue: '--- Select Storage ---',
         },
+        {
```
Comment (mareklibra) [inline src/components/Wizard/CreateVmWizard/StorageTab.js]: For the sake of maintainability, please change `renderConfigs` to function or map.

File: src/components/Wizard/CreateVmWizard/StorageTab.js [hunk 4]
```diff
@@ -372,6 +449,15 @@ class StorageTab extends React.Component {
           choices: this.props.storageClasses.map(getName),
           initialValue: '--- Select ---',
         },
+        undefined,
```
Comment (mareklibra) [inline src/components/Wizard/CreateVmWizard/StorageTab.js]: Please introduce constant for that, like `FIELD_READ_ONLY` or similar

File: src/components/Wizard/CreateVmWizard/StorageTab.js (hunk 2 above)
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/StorageTab.js]: Follow up https://github.com/kubevirt/web-ui-components/issues/131

File: src/components/Wizard/CreateVmWizard/StorageTab.js (hunk 3 above)
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/StorageTab.js]: Follow up https://github.com/kubevirt/web-ui-components/issues/132

File: src/components/Wizard/CreateVmWizard/StorageTab.js (hunk 4 above)
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/StorageTab.js]: Follow up https://github.com/kubevirt/web-ui-components/issues/133

File: src/components/Wizard/CreateVmWizard/BasicSettingsTab.js (hunk 1 above)
Comment (rawagner) [reply src/components/Wizard/CreateVmWizard/BasicSettingsTab.js]: removed

**Pull request merged**